### ISSUE TITLE
bug(MatCheckbox): hover area small than ripple

### ISSUE THREAD

**Issue by markkoo** [angular/components#19438]
#### Reproduction
https://stackblitz.com/edit/angular-japugn?file=src%2Fapp%2Fcheckbox-overview-example.html

Steps to reproduce:
1. Slowly hover to checkbox
2. When cursor touch the ripple area, it is still no able to check
 
 
#### Expected Behavior
When cursor touch the ripple area, it should be able the check. This behavior is follow Material IO Guide.

Material IO
![material IO](https://user-images.githubusercontent.com/4880701/82778478-1eede400-9e84-11ea-9e98-b37b08694727.gif)

Gmail
![google gmail](https://user-images.githubusercontent.com/4880701/82778500-2f9e5a00-9e84-11ea-9746-a4e6016d5b4e.gif)

Google Ads
![google ads](https://user-images.githubusercontent.com/4880701/82778516-34fba480-9e84-11ea-94db-96461febb854.gif)

#### Actual Behavior
Only when cursor touch the checkbox, then only can check.
![material angular ](https://user-images.githubusercontent.com/4880701/82778558-52c90980-9e84-11ea-9a45-45650f6b1081.gif)

Remark
Normally user needs a bigger space to check the checkbox, hope angular material team consider to follow material IO guide, even this hard to implement it.

#### Environment
Angular CLI: 9.1.6
Node: 12.16.1
OS: win32 x64

Angular: 9.1.9
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router
Ivy Workspace: Yes

Package                            Version
------------------------------------------------------------
@angular-devkit/architect          0.901.6
@angular-devkit/build-angular      0.901.6
@angular-devkit/build-ng-packagr   0.901.6
@angular-devkit/build-optimizer    0.901.6
@angular-devkit/build-webpack      0.901.6
@angular-devkit/core               9.1.6
@angular-devkit/schematics         9.1.6
@angular/cdk                       9.2.4
@angular/cli                       9.1.6
@angular/material                  9.2.4
@angular/material-moment-adapter   9.2.4
@ngtools/webpack                   9.1.6
@schematics/angular                9.1.6
@schematics/update                 0.901.6
ng-packagr                         9.0.0
rxjs                               6.5.5
typescript                         3.7.5
webpack                            4.42.0

**Comment by devversion** [comment]
Great issue. Thanks! This will be fixed by our MDC-based version of the Material Design checkbox. The checkbox currently lives in `@angular/material-experimental`, but at the current time of writing, this issue is already fixed there.

**Comment by amysorto** [comment]
Closing this issue since this is the behavior in the MDC based checkbox which is available soon in v15.
